gradle.properties

```diff
@@ -1,7 +1,7 @@
-version=2.0.5-SNAPSHOT
+version=2.1.0-SNAPSHOT
```

Comment:
I guess that is intended?

Reply:
Yes, because I want to release a new minor version